### DISCUSSION TITLE
📖 Looking for translators! - Feel free to open a pull request and add translations for your language

### DISCUSSION THREAD

**Discussion by PhantomWing** [PhantomWing/RusticDelight#6] [Announcements]
## 📖 Looking for translators! 📖
I would love to make the mod more complete by adding more language support. Google Translate only gets me so far, so feel free to contribute to the mod by adding more translation files, or possibly even updating any incorrect translations.

## Supported languages
Below is a list of all languages currently supported by Rustic Delight. A huge thank you to anyone who contributed!

✅ English
✅ Chilean Spanish (by vlaster666)
✅ Dutch
✅ German
✅ Japanese (by EndilCrafter)
✅ Kazakh (by ninsent)
✅ Korean (by TAILS)
✅ Russian (by MaxKuz67, JedaiGames and mpustovoi)
✅ Simplified Chinese (by Junnaturefox, ShangguanXi)
✅ Spanish (Argentina) (by ex0planet)
✅ Spanish (Chili) (by vlaster666)
✅ Taiwanese Mandarin (by Lobster0228)
✅ Vietnamese (by GodOfLac)

**Comment by vlaroca82** [comment]
[es_cl.json](https://github.com/user-attachments/files/25744536/es_cl.json)
Hi!
I translated the mod to Chilean Spanish (es_cl).

I'm attaching the language file here in case it's useful for the project or for other Spanish-speaking players.

Feel free to use or adapt it if needed. Thanks for the great mod!

my user is vlaster666

**Reply by PhantomWing** [reply]
Hi there! Thanks so much for translating, I'll make sure it's added to Rustic Delight. I'm releasing a new version for NeoForge 1.21.1 today where the file will be included :)
